src/test/java/edu/oakland/test/database/DatabaseGisManagerTest.java

```diff
@@ -0,0 +1,61 @@
+package edu.oakland.test.database;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.production.database.DatabaseGisManagerImplementation;
+import edu.oakland.test.database.DatabasePersistentStorageStub;
+import java.lang.IllegalArgumentException;
+import java.time.LocalDateTime;
+import java.util.Random;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Database GisInterface Test")
```

Comment:
For sake of standardization to make it easier to read and search the logs, can we change this to `DatabaseGisManagerImplementation Unit Tests`? Thank you!
```suggestion
@DisplayName("DatabaseGisManagerImplementation Unit Tests")
```